test/evergreen.yml

```diff
@@ -611,6 +611,15 @@ functions:
           ${test_env_vars|} ${python_binary} ../../../bench/perf_run_py/perf_run.py --${test_type|wtperf} -e ${exec_path|./wtperf} -t ${perf-test-path|../../../bench/wtperf/runners}/${perf-test-name} -ho WT_TEST -m ${maxruns} -g "../.." -v -i "$JSON_TASK_INFO" -b -o test_stats/evergreen_out_${perf-test-name}.json ${wtarg}
           ${test_env_vars|} ${python_binary} ../../../bench/perf_run_py/perf_run.py --${test_type|wtperf} -e ${exec_path|./wtperf} -t ${perf-test-path|../../../bench/wtperf/runners}/${perf-test-name} -ho WT_TEST -m ${maxruns} -g "../.." -v -i "$JSON_TASK_INFO" -re -o test_stats/atlas_out_${perf-test-name}.json ${wtarg}
 
+  "smoke test":
```

Comment:
I'd prefer the function name to reflect the scenario of the tests (abort test) instead of the name of the test programs (smoke test). The only non-abort test seems to be the `random-directio`, which could be excluded from this refactor change?

Reply:
I am trying to think of a different name... I am not sure naming it `abort test` is great, as you said we have `random-directio` and `test_truncated_log`

Reply:
Would `csuite smoke test` work?

Reply:
Sounds a little better.

Reply:
I renamed it to `csuite smoke test` in 24af2120730fb5eadff177ced2b5164a7938c4b5.